contracts/core/src/solve/Solve.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity =0.8.24;
+
+library Solve {
+    enum Status {
+        Open,
+        Accepted,
+        Cancelled,
+        Rejected,
+        Fulfilled,
+        Paid
+    }
+
+    /**
+     * @notice A request to execute a call on another chain, backed by a deposit.
+     * @param id            ID for the request, globally unique per inbox.
+     * @param updatedAt     Timestamp request status was last updated.
+     * @param from          Address of the user who created the request.
+     * @param fulfilledBy   Address of the solver that fulfilled the request.
+     * @param status        Request status (open, accepted, cancelled, rejected, fulfilled, paid).
+     * @param call          Details of the call to be executed on another chain.
+     * @param deposits      Array of deposits backing the request.
+     */
+    struct Request {
+        bytes32 id;
+        uint96 updatedAt;
```

Comment:
uint40 is all you need for a timestamp as it will work until the year 3593. The smaller variable will also let you move Status into the same slot.